WolvenKit.Modkit/RED4/RedConverter.cs

```diff
@@ -58,7 +60,16 @@ public bool ConvertToAndWrite(ETextConvertFormat format, string infile, Director
             try
             {
                 var text = ConvertToText(format, infile);
-                var outpath = Path.Combine(outputDirInfo.FullName, $"{Path.GetFileName(infile)}.{format}");
+                var destination = Path.GetDirectoryName(infile);
+                // If DepotPath structure exists then add it to output directory
+                // Would be nice to make this a runtime option like --structure "[none|depotpath]" or something
+                if (destination.IndexOf(@"\\source\\archive\\") > 0)
+                {
+                    destination = Regex.Split(destination, "\\source\\archive\\")[1];
+                    destination = Path.Combine(outputDirInfo.FullName, destination);
+                    Directory.CreateDirectory(destination);
+                }
+                var outpath = Path.Combine(destination, $"{Path.GetFileName(infile)}.{format}");
```

Comment:
This entire thing can be done with:

```csharp
var outDir = Path.GetDirectoryName(infile);
var idx = outDir.IndexOf(@"\source\archive");
if (idx != -1)
{
    outDir = Path.Combine(outputDirInfo.FullName, outDir.Substring(idx + 16));
    Directory.CreateDirectory(outDir);
}
var outpath = Path.Combine(outDir, $"{Path.GetFileName(infile)}.{format}");
```
Again, you'd have to check for the actual directory separator, and not `\` though. so some formatting for that would have to be added